bundle/Resources/config/services/mappers.yml

```diff
@@ -3,7 +3,7 @@ services:
         class: Netgen\Bundle\eZPlatformAdvancedSearchBundle\Core\ItemFilter\ValueMappers\FacetTitleMapper
         arguments:
             - '@eztags.api.service.tags'
-            - '@ezpublish.translation_helper'
+            - \\Ibexa\\Core\\Helper\\TranslationHelper
```

Comment:
Needs to be `@Ibexa\\Core\\Helper\\TranslationHelper`

Reply:
Fixed